app/assets/stylesheets/home/components/psychologist_page/psychologist_page.scss

```diff
@@ -0,0 +1,109 @@
+@import '../variables';
+
+.psychologist-info{
+  align-items: center;
+  @include flex-column;
+
+  .psychologist-info-top{
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+
+    .psychologist-info_img{
+      border: 1px solid blue;
+      border-radius: 95px;
+      margin: 20px 0;
+      width: 50%;
+    }
+  
+    .psychologist-info-about{
+      border: 3px solid #C6E0EF;
+      border-radius: 20px;
+      color: $text-color;
+      padding: 10px;
+      margin-bottom: 20px;
+      width: 85%;
+  
+      .psychologist-info-about_muted{
+        font-weight: normal;
+      }
+  
+      p{
+        color: #000; 
+        font-weight: bold;
+      }
+  
+      .psychologist-info-about_txtarea{
+        background-color: #66C0DD;
+        border-radius: 20px;
+        height: 200px;
+        margin: 20px 0;
+      }
+    }
+  }
+  
+  .psychologist-info-profesional{
+    background-color: #66C0DD;
+    border-radius: 20px;
+    color: #fff;
+    margin-bottom: 20px;
+    padding: 30px 30px;
+    width: 85%;
+
+    .psychologist-info-profesional_text{
+      margin-bottom: 30px;
+    }
+
+    span{
+      font-weight: bold;
+    }
+  }
+}
+
+@media(min-width: $breakpoint-desktop){
+  .psychologist-info{
+    position: inherit;
+    height: 50vh;
+
+    .psychologist-info-top{
+      flex-direction: row;
+      gap: 70px;
+      margin-top: 30px;
```

Comment:
If you already have the class name on the parent selector you can only do something like this:
```
&-top{...}
```
This automatically prepends the parent name to this sector when SASS compiles the code
you can find more info [here](https://sass-lang.com/documentation/style-rules/parent-selector)